contracts/reward/contract/KlaytnReward_test.go

```diff
@@ -37,45 +38,44 @@ func TestSmartContract(t *testing.T) {
 	key2, _ := crypto.GenerateKey()
 	auth2 := bind.NewKeyedTransactor(key2)
 
-	alloc := blockchain.GenesisAlloc{auth.From: {Balance: big.NewInt(params.KLAY)}, auth2.From: {Balance: big.NewInt(params.KLAY)}}
+	initialValue := big.NewInt(params.KLAY)
+	withdrawValue := big.NewInt(500000000)
+
+	alloc := blockchain.GenesisAlloc{auth.From: {Balance: initialValue}, auth2.From: {Balance: initialValue}}
 	sim := backends.NewSimulatedBackend(alloc)
 
 	// Deploy a token contract on the simulated blockchain
 	_, _, reward, err := DeployKlaytnReward(auth, sim)
 	if err != nil {
 		log.Fatalf("Failed to deploy new token contract: %v", err)
 	}
-	// Print the current (non existent) and pending name of the contract
-	tx, err := reward.Reward(&bind.TransactOpts{From: auth.From, Signer: auth.Signer, Value: big.NewInt(500000000)}, auth2.From)
+	sim.Commit()
```

Comment:
May I ask why this `Commit` is necessary?

Reply:
When `reward.Reward` is called, the function creates a tx with an estimated gas. In the previous code, the estimation was executed base on a pending block and worked well without this line (a contract deploying tx created by `DeployKlaytnReward` was in the pending block).
With this change, however, the estimation is executed against the latest block and doesn't work well with the pending state. So, I put this line to commit the contract deploying tx.

Reply:
Thanks for your explanation. :)
Checking if the contract was deployed well would be nice.